### PR TITLE
Stuart Kilgore

### PR DESCRIPTION
**Pull request by stu-k**
# Checkpoint Rubric

This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

## Checkpoint 1: Company/Organization Website following Wireframe
- [x] A CSS file is effectively linked to the HTML file in the Head section: 10 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 10 points 
- [ ] The webpage links to another page (multi-page website): 10 points
- [x] The company/orgnanization name is lined up with the navigation menu at the top of the page: 10 points
- [x] The title is centered in front of a background image: 10 points
- [x] The body images are sized and formatted according to the wire frame: 10 points
- [x] The body text is sized, positioned, and formatted to match the fire frame: 10 points
- [x] Flexbox or floats are used to accurately replicate the layout of the wireframe: 10 points
- [x] The footer consists of a background image, centered text, and 2 buttons that function as links: 10 points
- [x] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 10 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 2: Foundry Winery Website
- [x] Responsive navbar: 20 points
- [x] Responsive site (resizes to fit desktop, tablet, and mobile views): 30 points 
- [x] Matched layout of original site: 20 points
- [x] Matched font of original site: 10 points
- [x] Properly commented code: 10 points
- [x] Properly indented code: 10 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 3: Learning Team Project
- [ ] Mockup meets customer requirements: 25 points
- [ ] Website meets customer requirements: 25 points
- [ ] Learning team worked together effectively and contributed equally: 25 points
- [ ] Class presentation was engaging and effective: 25 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 4: Personal Portfolio Website
- [ ] Responive: 20 points
- [ ] Evidence of Effective Use of Design Principles: 20 points
- [ ] Links to Resume: 20 points
- [ ] Links to at least 3 Projects: 20 points
- [ ] Effective Communicates Desired Message ("Hire me"): 20 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

### CODE REVIEW

File: 01week/bio/index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      what little HTML I learned before is quickly coming back to me.
+      I have a passion for learning, be it coding, languages, or how to
+      better myself, I'm always excited to learn! I'm an avid gamer, and
+      double-dog-dad.
+    </p>
+    <img src="dogs-couch.JPG" alt="dogs" class="pic pic-dogs">
+    <p class="bio">
+      These are my dogs, Pi (left) and Churchill (right). Churchill likes
+      to stare at my guests until they pet him, while Pi licks the air
+      when she's excited. <strong>WHY WON'T THIS PICTURE COOPERATE</strong>
```
Comment (egarcia1868) [inline 01week/bio/index.html:31]: I noticed your dogs pic is sideways.  Did you try to open the pic in an image program, turning it, and then saving it again?

Comment (stu-k) [reply 01week/bio/index.html:31]: I did! And the weird thing is that the picture doesn't display as sideways when viewed from a mobile device.

Comment (egarcia1868) [reply 01week/bio/index.html:31]: oh, weird...

File: 01week/bio/css/style.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+/* Some style for the bio sections */
+.bio {
+  background-color: #92b280;
+  border-radius: 10px;
+  color: #ede5ad;
+  font-weight: 300;
+  text-align: center;
+  margin: auto 37% 50px 37%;
+}
```
Comment (egarcia1868) [inline 01week/bio/css/style.css:41]: Don't see anything off on this code.  Nice!

File: 01week/bio/index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    </p>
+<!-- A picture of my dogs with a bio -->
+    <img src="dogs-couch.JPG" alt="dogs" class="pic pic-dogs">
+    <p class="bio">
+      These are my dogs, Pi (left) and Churchill (right). Churchill likes
+      to stare at my guests until they pet him, while Pi licks the air
+      when she's excited. <strong>WHY WON'T THIS PICTURE COOPERATE</strong>
+    </p>
+  </body>
+</html>
```
Comment (egarcia1868) [inline 01week/bio/index.html:34]: Everything looks good on this one also.

File: 01week/resume/css/style.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+/* Education */
+.edu-header {
+  text-align: center;
+}
+
+.edu-info {
+  text-align: left;
+}
```
Comment (egarcia1868) [inline 01week/resume/css/style.css:95]: Aaaaaaaaaaaall good

File: 01week/resume/index.html [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+                  <li>
+                    Happy Bar - Boulder, Co
+                  </li>
+                </ul>
+              </ul>
+        </div>
+    </main>
+</body>
+
+</html>
```
Comment (egarcia1868) [inline 01week/resume/index.html:222]: Didn't see anything that I noticed could be improved.

File: 02week/nasa/css/style.css [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+.send {
+  align-items: center;
+  background-color: #424242;
+  color: white;
+  display: flex;
+  font-weight: bold;
+  height: 25px;
+  justify-content: center;
+  width: 80px;
+}
```
Comment (egarcia1868) [inline 02week/nasa/css/style.css:52]: All good.

File: 02week/nasa/index.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+        <input class="field-name" type="text" name="" value="">
+        <p>Message</p>
+        <textarea class="field-message" name="name" rows="5" cols="35">
+        </textarea>
+        <p class="send">send</p>
+      </div>
+    </div>
+    </footer>
+  </body>
+</html>
```
Comment (egarcia1868) [inline 02week/nasa/index.html:53]: Again, all good

File: index.html [hunk 8]
```diff
@@ -1,2 +1,22 @@
...
+  <body>
+<!-- Links to other pages used for class in ACA -->
+    <div class="main">
+      <h1>
+        Stu.
+      </h1>
+      <a href="01week/resume" class="link">Resume</a>
+      <br />
+      <a href="01week/bio" class="link">Bio</a>
+      <br />
```
Comment (egarcia1868) [inline index.html]: the " /"s after the line breaks on 16 and 18 are unnecessary.

File: style.css [hunk 9]
```diff
@@ -1 +1,17 @@
...
+  background-color: black;
+  border: 5px solid black;
+  border-radius: 10px;
+  margin: 15% 40%;
+  text-align: center;
+}
+
+.link {
+  font-weight: 400;
+}
```
Comment (egarcia1868) [inline style.css]: All good.

**Pull request closed without merging**